onadata/apps/api/tests/viewsets/test_user_profile_viewset.py

```diff
@@ -1627,19 +1622,14 @@ def test_cache_isolation_between_users_retrieve(self):
         self.assertEqual(cache_b_a["first_name"], "UserA")
         self.assertTrue("email" not in cache_b_a)
 
-        # anonymous user retrieves user A's profile
+        # anonymous user retrieves user A's profile - should be blocked
```

Comment:
@FrankApiyo Since anonymous users are now blocked, do we need test cases for `anonymous user retrieves user A's profile - should be blocked`? Seems test `test_profiles_get_anon` is sufficient